package.json

```diff
@@ -48,11 +48,11 @@
         "grunt-contrib-watch": "0.6.1",
         "grunt-targethtml": "0.2.6",
         "grunt-usemin": "0.1.11",
```

Comment:
@zaggino You seem to have gone back here in https://github.com/zaggino/brackets-electron/commit/d04ac79e912f9c9775576ab99459122af1576094#diff-5ca1cd5bde62416d7f8469f969948a94R73.
Is there a specific reason or can we also update it?